qdrant_client/http/models/models.py

```diff
@@ -2232,6 +2326,10 @@ def __str__(self) -> str:
     SparseIndexTypeOneOf1,
     SparseIndexTypeOneOf2,
 ]
+StartFrom = Union[
+    StrictFloat,
+    datetime,
+]
```

Comment:
https://github.com/qdrant/pydantic_openapi_v3/issues/24

Reply:
we want to accept integers here as well, it will work with pydantic v2 https://github.com/pydantic/pydantic/issues/7504
however, it won't work with pydantic v1 which will crash, we need to add `StrictInt`

Reply:
Already added to the spec with qdrant/qdrant#3726. In the meantime I will add it manually to integrate it to the PR.

Reply:
Solved